### DISCUSSION TITLE
not Work in OBS

### DISCUSSION THREAD

**Discussion by kasebgamer** [DistroAV/DistroAV#701] [Q&A - Support]
I followed all the tutorials
And even fully installed
But it does not work on obs
![ndi](https://user-images.githubusercontent.com/94053325/141301566-b6c400fd-c8b8-42e1-9abd-d954c5923862.PNG)

**Comment by Trouffman** [comment]
Closing old thread.
Reason : Using an old plugin/OBS version or old-issue and no debug info provided.
